.github/workflows/ci_e2e-custom.yaml

```diff
@@ -22,6 +22,12 @@ jobs:
                 php: [ "8.1" ]
                 symfony: [ "^6.3" ]
                 mysql: [ "8.0" ]
+                resource: [ "^1.9" ]
+                include:
+                    -   php: "8.1"
+                        symfony: "^6.3"
+                        mysql: "8.0"
+                        resource: "^1.11@dev"
```

Comment:
I'm just wondering if now we prefer to test the dev or the alpha release.
I think dev is better to test it before new releases.